test/fixtures/lint-dirty-files/updated-bad.scss

```diff
@@ -0,0 +1,4 @@
+     #stuff
+{display: block
+;;
+   }
```

Comment:
another nitpick: please make sure your editor is using [EditorConfig](http://editorconfig.org) - thanks!